README.md

```diff
@@ -11,7 +11,7 @@
 A [Terraform](https://www.terraform.io) module to create a [Google Cloud Storage](https://cloud.google.com/storage) on [Google Cloud Services (GCP)](https://cloud.google.com/).
 
 **_This module supports Terraform version 1
-and is compatible with the Terraform Google Provider version 4._** and 5._**
+and is compatible with the Terraform Google Provider version 5._**
```

Comment:
@Esanim the README should be clear what versions are supported. This should say that it's 5.10 and later.